### PR TITLE
Update zh-cn.json

### PR DESCRIPTION
**Pull request by steve02081504**
- Remove the carriage return from Key so that I18N will work properly in some browsers.
- Append two HTML files that will not actually be utilized so that the I18N tool will work properly.

## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Cohee1207** [comment]
> Append two HTML files that will not actually be utilized so that the I18N tool will work properly.

@Wolfsblvt what do you think of this? It looks like a workaround to satisfy the i18n extraction tool, but it is not entirely bad.

**Comment by Wolfsblvt** [comment]
A bit weird, but I don't think its too bad. Just makes it even harder to keep them up-to-date if there are changes to the actual source file. But if translators keep an eye out for that, and devs have to worry less, I don't mind.
Good idea, generally.

I would suggest naming them `xxxxx.i18n.html`, with `xxxxx` being the name of the source file. If the export tool still picks that up. Makes it more clear that this is a supplementary file.

File: public/index.html [hunk 1]
```diff
@@ -4275,7 +4275,7 @@ <h4 data-i18n="STscript Settings">STscript Settings</h4>
...
+                                    <label class="checkbox_label" title="Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated.&NewLine;e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;&NewLine;&NewLine;(This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)" data-i18n="[title]Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated. e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;  (This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)">
```
Comment (Wolfsblvt) [inline public/index.html]: If you change the i18n string to be different from the exact content of the value, you likely should use a shortened key here. Like we already do in many other places. `stscript_parser_flag_replace_getvar_label` or something.

File: public/scripts/extensions/gallery/index.js [hunk 2]
```diff
@@ -406,7 +407,7 @@ SlashCommandParser.addCommandObject(SlashCommand.fromProps({ name: 'show-gallery
         showCharGallery();
         return '';
     },
-    helpString: 'Shows the gallery.',
+    helpString: translate('Shows the gallery.'),
```
Comment (Cohee1207) [inline public/scripts/extensions/gallery/index.js]: Let's maybe handle slash commands some other time. I'm not sure what would be the ultimate way to approach them.

File: public/index.html (hunk 1 above)
Comment (steve02081504) [reply public/index.html]: fixed in [`3e5f982` (#2551)](https://github.com/SillyTavern/SillyTavern/pull/2551/commits/3e5f98223d0c45ea1a94f793bcc075c84ef470cd)

File: public/scripts/extensions/gallery/index.js (hunk 2 above)
Comment (steve02081504) [reply public/scripts/extensions/gallery/index.js]: fixed in [`3e5f982` (#2551)](https://github.com/SillyTavern/SillyTavern/pull/2551/commits/3e5f98223d0c45ea1a94f793bcc075c84ef470cd)

**Review by Wolfsblvt — APPROVED** [review]
Looks good.

**Review by Cohee1207 — APPROVED** [review]

**Pull request merged**